### PR TITLE
[Interpreter] Support group arg for Convolution.

### PR DESCRIPTION
**Pull request by beicy**
Support group arg for quantized I8 implementation of Convolution.

### CODE REVIEW

File: tests/unittests/quantizationTest.cpp [hunk 1]
```diff
@@ -144,7 +144,7 @@ static std::pair<Function *, SaveNode *>
 createSimpleGraphForQuantization(Module *M) {
   Function *F = M->createFunction("main");
 
-  auto *A = F->getParent()->createVariable(ElemKind::FloatTy, {1, 32, 32, 3},
```
Comment (nadavrot) [inline tests/unittests/quantizationTest.cpp]: Why did you change this test?

File: tests/unittests/quantizationTest.cpp [hunk 2]
```diff
@@ -154,7 +154,7 @@ createSimpleGraphForQuantization(Module *M) {
                                            Variable::TrainKind::None);
   fillStableRandomData(B->getHandle(), 2001, 1);
 
-  ConvolutionNode *CV = F->createConv("conv", A, 16, 5, 1, 2, 1);
```
Comment (nadavrot) [inline tests/unittests/quantizationTest.cpp]: Why did you change this test?

File: lib/Backends/Interpreter/InterpreterNodes.cpp [hunk 3]
```diff
@@ -111,6 +111,11 @@ void Interpreter::fwdConvolutionInst_I8Impl(Value *inV, Value *outV,
   ShapeNHWC odim(outW.dims());
   ShapeNHWC idim(inW.dims());
 
+  assert(idim.c % group == 0 && "Input channels must be divisible by group.");
+  assert(odim.c % group == 0 && "Output channels must be divisible by group.");
+  size_t inCperG = idim.c / group;
+  size_t outCperG = odim.c / group;
```
[the remark this answers is not in the document]
Comment (beicy) [reply lib/Backends/Interpreter/InterpreterNodes.cpp]: Then this assert should be added in Function::createConv() I think. It seems to me this is first place the group is used as divisor.

File: tests/unittests/quantizationTest.cpp (hunk 1 above)
Comment (beicy) [reply tests/unittests/quantizationTest.cpp]: To test the group arg > 1.

File: lib/Backends/Interpreter/InterpreterNodes.cpp [hunk 4]
```diff
@@ -130,57 +135,58 @@ void Interpreter::fwdConvolutionInst_I8Impl(Value *inV, Value *outV,
   // multiplication part of the calculation.
   float matMulScale = inScale * filterScale;
 
-  size_t inChannels = idim.c;
-
   // For each input in the batch:
   for (size_t n = 0; n < idim.n; n++) {
+    // For each group of input channels:
+    for (size_t g = 0; g < group; g++) {
+      // For each layer in the output tensor:
```
Comment (rdzhabarov) [inline lib/Backends/Interpreter/InterpreterNodes.cpp]: from the fp version:
 // For each output channel in the group:
      for (size_t d = g * outCperG; d < (g + 1) * outCperG; d++) {

**Comment by beicy** [comment]
@rdzhabarov  Not yet. My next task :)

File: tests/unittests/quantizationTest.cpp (hunk 2 above)
Comment (beicy) [reply tests/unittests/quantizationTest.cpp]: To test group arg > 1. Since Input channels should be divisible by group, I modified both of them to 2.
